non-pinned.json

```diff
@@ -13,6 +13,9 @@
   "prConcurrentLimit": 0,
   "prHourlyLimit": 0,
   "branchConcurrentLimit": 0,
+  "minimumReleaseAge": "7 days",
```

Comment:
❓ Per examples like https://docs.renovatebot.com/configuration-options/#prevent-holding-broken-npm-packages what if we considered this per manager?